crates/database/src/orders.rs

```diff
@@ -730,7 +762,6 @@ pub struct OrderWithQuote {
 
 pub async fn user_orders_with_quote(
     ex: &mut PgConnection,
-    min_valid_to: i64,
```

Comment:
This was unused, does it make sense to add it to the query or to delete it as I did? :thinking: